client/commonFramework/end.js

```diff
@@ -112,6 +112,7 @@ $(document).ready(function() {
         common.displayTestResults(tests);
         if (solved) {
           common.showCompletion();
+					common.codeStorage.removeStoredValue(common.challengeName);
```

Comment:
It is too right to be cool. I was trying to be funny, I mean it should be indented to left 8/10 spaces. Same for line 56 maybe too?